src/operators.jl

```diff
@@ -88,22 +89,128 @@ tensor(op::AbstractOperator) = op
 tensor(operators::AbstractOperator...) = reduce(tensor, operators)
 
 
+
+"""
+    check_indices(indices::Array)
+
+Determine whether a collection of indices, written as a list of (integers or lists of integers) is unique.
+This assures that the embedded operators are in non-overlapping subspaces.
+"""
+function check_indices(indices::Array)
```

Comment:
It would make sense to move this function into the `sortedindices` module

Reply:
great